File: src/main/java/org/dbtools/query/sql/MysqlQueryBuilder.java
/*
 * PostgresqlQueryBuilder.java
 *
 * Created on October 27, 2007
 *
 * Copyright 2007 Jeff Campbell. All rights reserved. Unauthorized reproduction 
 * is a violation of applicable law. This material contains certain 
 * confidential or proprietary information and trade secrets of Jeff Campbell.
 */

package org.dbtools.query.sql;

import org.dbtools.query.shared.QueryUtil;

/**
 *
 * @author Jeff
 */
public class MysqlQueryBuilder extends SQLQueryBuilder {

    @Override
    public String formatLikeClause(String column, String value) {
        return QueryUtil.formatLikeClause(column, value);
    }
    
    @Override
    public String formatIgnoreCaseLikeClause(String column, String value) {
        return formatLikeClause(column, value);
    }
    
    
}

File: dbtools-query/src/commonMain/kotlin/org/dbtools/query/sql/MysqlQueryBuilder.kt
/*
 * PostgresqlQueryBuilder.java
 *
 * Created on October 27, 2007
 *
 * Copyright 2007 Jeff Campbell. All rights reserved. Unauthorized reproduction 
 * is a violation of applicable law. This material contains certain 
 * confidential or proprietary information and trade secrets of Jeff Campbell.
 */
package org.dbtools.query.sql

import org.dbtools.query.shared.util.QueryUtil

/**
 *
 * @author Jeff
 */
class MysqlQueryBuilder : SQLQueryBuilder() {
    override fun formatLikeClause(field: String, value: String): String {
        return QueryUtil.formatLikeClause(field, value)
    }

    override fun formatIgnoreCaseLikeClause(field: String, value: String): String {
        return formatLikeClause(field, value)
    }
}
